src/mainwindow.cpp

```diff
@@ -100,6 +112,85 @@ void MainWindow::updateBoms(DbManager *d)
     }
 }
 
+bool MainWindow::refillMatchFromSaveFileToDatabase(const std::string& path)
+{
+    // Read and stock all clubs
+    std::ifstream matchFlow(path.c_str(), std::ios::in);
```

Comment:
You can use QTextStream instead ;)

Reply:
Not the easy part of this code review but I sincerely think that it is clearer now!